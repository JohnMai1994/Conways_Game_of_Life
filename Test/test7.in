new 10
init 
3 3 5 3
3 4 4 4 4 5
-1 -1
print
step
print
step
print
step
print
step
print
step
print
step
print
step
