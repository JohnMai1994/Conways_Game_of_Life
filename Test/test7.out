__________
__________
__________
___X_X____
___XX_____
____X_____
__________
__________
__________
__________
__________
__________
__________
___X______
___X_X____
___XX_____
__________
__________
__________
__________
__________
__________
__________
____X_____
__XX______
___XX_____
__________
__________
__________
__________
__________
__________
__________
___X______
__X_______
__XXX_____
__________
__________
__________
__________
__________
__________
__________
__________
__X_X_____
__XX______
___X______
__________
__________
__________
__________
__________
__________
__________
__X_______
__X_X_____
__XX______
__________
__________
__________
__________
__________
__________
__________
___X______
_XX_______
__XX______
__________
__________
__________
